project.clj

```diff
@@ -142,11 +142,13 @@
    [pretty "1.0.4"]                                                   ; protocol for defining how custom types should be pretty printed
    [prismatic/schema "1.1.11"]                                        ; Data schema declaration and validation library
    [redux "0.1.4"]                                                    ; Utility functions for building and composing transducers
+   [reifyhealth/specmonstah "2.0.0"]                                  ; Generate fixtures to test huge databases
    [riddley "0.2.0"]                                                  ; code walking lib -- used interally by Potemkin, manifold, etc.
    [ring/ring-core "1.8.0"]
    [ring/ring-jetty-adapter "1.8.1"]                                  ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
    [ring/ring-json "0.5.0"]                                           ; Ring middleware for reading/writing JSON automatically
    [stencil "0.5.0"]                                                  ; Mustache templates for Clojure
+   [talltale "0.4.3"]                                                 ; Generate realistic data for fixtures
```

Comment:
since these aren't used in prod, I wouldn't put the dependencies here -- put them in `:dev` instead. No need to bloat the size of the JAR we ship to people with stuff we're not using in prod